### DISCUSSION TITLE
📢 Update to our Data Usage Policy

### DISCUSSION THREAD

**Discussion by AnanyaJha** [forcedotcom/Einstein-GPT-for-Developers#51] [Announcements]
With the latest release of Einstein for Developers, we've revamped our Data Usage Policy. We **do not** use any customer code or feedback to help train our internal LLM, CodeGen. With this change, _new_ users will no longer be required to `Enable` the Einstein for Developers product via any Setup pages - you will be enabled by default.

Learn more in our updated [Terms & Conditions
](https://developer.salesforce.com/tools/vscode/en/einstein/einstein-termsofuse)
If this policy changes in the future, customers will be required to _opt in_ to new data usage guidelines. If you have questions or concerns, please share them below.

**Comment by sirephil** [comment]
How can we ensure that all our devs are automatically opted out and ideally not include the copilot installation?

Our business currently disallows all use of AI with our source code, and while this change means you will not train your AI assistant with our code, this still violates our corporate policy.

**Comment by AnanyaJha** [comment]
Hi @sirephil ! I understand, we recommend that you work with your development teams to ensure that they don't install the Einstein for Developers extension from the VS Code Marketplace. Your developers will only have access to this tool if they manually install the Einstein for Developers extension via the VS Code or OpenVSX Marketplaces.

**Comment by dgyip** [comment]
Hello @AnanyaJha can you clarify the data usage policy as it relates to the Einstein for Developers FAQ here? https://developer.salesforce.com/tools/vscode/en/einstein/einstein-faq
According to the FAQ, it states _"Some of your code and entity schema can be used to improve Einstein for Developers and train CodeGen."_
This seems to contradict the statements in the Terms and Conditions link you shared above.  Is the FAQ out-of-date or is there a nuance that I am not understanding?

**Comment by AnanyaJha** [comment]
Hi @dgyip ! I'm not seeing that sentence in our FAQs anymore, but please let me know if you see confusing language elsewhere! At this point, Salesforce does not use any customer info to train the CodeGen models.
